src/constants/translations/en/tutor-home-page.json

```diff
@@ -0,0 +1,46 @@
+{
+    "findStudentBlock": {
+      "title": {
+        "student": "A Global Network of Tutors",
+        "tutor": "A Global Network of Students"
+      },
+      "description": "A Space2Study platform is a place where you can learn from the best. Type what would you like to teach and find students for that.",
+      "label": "What would you like to teach?",
+      "button": "Find student"
+    },
+    "popularCategories": {
+      "title": "Popular Categories",
+      "description": "Explore tutoring categories you're passionate about.",
+      "viewMore": "View more"
+    },
+    "howItWorks": {
+      "title": "How it works",
+      "description": "4 steps to easy and funny learning with Space2Study",
+      "selectATutor": {
+        "title": "Select a Tutor",
```

Comment:
remove all unnecessary translations. I see that some of them you just copied from the student's page and didn't even change it for the tutor

Reply:
> remove all unnecessary translations. I see that some of them you just copied from the student's page and didn't even change it for the tutor

removed it